### PR TITLE
python exercises

### PR DESCRIPTION
**Pull request by miller9**
review the python exercises

### CODE REVIEW

File: 10.py [hunk 1]
```diff
@@ -1,36 +1,39 @@
...
 	return False
 
 
-list_1=["python",963,'abc',321,954810,3]
-list_2=[12,"py",'bwqeqweqwc',9,1,2,4,5,6,7,8,963]
-ans=overlapping(list_1,list_2)
-print ("Have both lists at least 1 member in common?",ans)
+list_1 = ["python",963,'abc',321,954810,3]
+list_2 = [12,"py",'bwqeqweqwc',9,1,2,4,5,6,7,8,963]
+ans = overlapping(list_1,list_2)
```
Comment (PabloVallejo) [inline 10.py]: Add spaces after `list_1,`

File: 10.py [hunk 2]
```diff
@@ -1,36 +1,39 @@
...
-		i+=1
+		i += 1
 	return False
 
 
-list_1=["python",963,'abc',321,954810,3]
-list_2=[12,"py",'bwqeqweqwc',9,1,2,4,5,6,7,8,963]
-ans=overlapping(list_1,list_2)
-print ("Have both lists at least 1 member in common?",ans)
+list_1 = ["python",963,'abc',321,954810,3]
```
Comment (PabloVallejo) [inline 10.py]: Add spaces after coma

File: 13_b.py [hunk 3]
```diff
@@ -1,20 +1,22 @@
...
 	13. The function max() from exercise 1) and the function max_of_three() from exercise 2) 
 	will only work for two and three numbers, respectively. 
 	But suppose we have a much larger number of numbers, 
 	or suppose we cannot tell in advance how many they are?
 	Write a function max_in_list() that takes a list of numbers and returns the largest one.
+
 	""")
 
-def max_in_list(l):
+def max_in_list (l):
```
Comment (PabloVallejo) [inline 13_b.py]: Remove space after function name

File: 13_b.py [hunk 4]
```diff
@@ -1,20 +1,22 @@
...
-	print ("The original list is:",l)				# show the original list
+	print ("The original list is:", l)				# show the original list
 	l.sort()										# If the list is ordered, last value is the greatest one
-	print ("The ordered list is :",l)				# show the ordered list
-	print ("Greatest value is   :",l[(len(l)-1)])	# show the greatest value (last value)
+	print ("The ordered list is :", l)				# show the ordered list
+	print ("Greatest value is   :", l[ (len(l) - 1) ])	# show the greatest value (last value)
 
-l1=[234,-300,1,213,576,678,3456,4646,-13455,-98765,1324,876543,982,98,76]
+l1 = [234,-300,1,213,576,678,3456,4646,-13455,-98765,1324,876543,982,98,76]
```
Comment (PabloVallejo) [inline 13_b.py]: Add space after comas

File: 14.py [hunk 5]
```diff
@@ -1,17 +1,20 @@
...
+		int_list.append( len(word) )
 
-	print ("\nThe list of words is   :",words_list)
-	print ("The list of integers is:",int_list)
+	print ("\nThe list of words is   :", words_list)
+	print ("The list of integers is:", int_list)
 	print ()
 
-l1=["david","gilmour","roger","waters","ray","manzarek","jim","morrison"]
+l1 = ["david","gilmour","roger","waters","ray","manzarek","jim","morrison"]
```
Comment (PabloVallejo) [inline 14.py]: Add space after comas

File: 15.py [hunk 6]
```diff
@@ -1,23 +1,25 @@
...
+		if (len(words_list[i]) == max_value):
+			print ("The list of words is 	  :", words_list)
+			print ("The longest word is 	  :", words_list[i])
+			print ("The length of the word is :", max_value)
 			break
-		i+=1
+		i += 1
 	print ()
 
 l1 = ["high_hopes","echoes","comfortably_numb","johnny_be_good","roll_over_bethoven","school_days"]
```
Comment (PabloVallejo) [inline 15.py]: Add space after comas

File: 16.py [hunk 7]
```diff
@@ -1,16 +1,19 @@
...
+		if (len(word) > n):
 			new_list.append(word)
 
-	print ("The original list of words is	   :",words_list)
-	print ("The list of words longer than",n,"is :",new_list)
+	print ("The original list of words is	   :", words_list)
+	print ("The list of words longer than", n, "is :", new_list)
 	print ()
 
 l1 = ["deep","down","in","louisiana","close","to","new","orleans"]
```
Comment (PabloVallejo) [inline 16.py]: Add space after comas

File: 18.py [hunk 8]
```diff
@@ -1,6 +1,40 @@
...
+	8. Pack my box with five dozen liquor jugs.
+	9. Jinxed wizards pluck ivy from the big quilt
 
-say_hi()
\ No newline at end of file
+	""")
+
+def is_pangram ():
+
+	abc = ['a','b','c','d','e','f','g','h','i','j','k','l','m','n','o','p','q','r','s','t','u','v','w','x','y','z']
```
Comment (PabloVallejo) [inline 18.py]: add space after comas

File: 20.py [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+	That is, write a function `translate()` that takes a list of 
+	English words and returns a list of Swedish words.
+
+
+	""")
+
+def translate (list):
+
+	output = ""
+	b_lexicon = { "merry":"god", "christmas":"jul", "and":"och", "happy":"gott", "new":"nytt", "year":"år" }
```
Comment (PabloVallejo) [inline 20.py]: Add space after `:` colon

File: 22.py [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+	22. In cryptography, a Caesar cipher is a very simple encryption techniques
+	in which each letter in the plain text is replaced by a letter some
+	fixed number of positions down the alphabet. 
+	For example, with a shift of 3, A would be replaced by D, 
+	B would become E, and so on. The method is named after Julius Caesar, 
+	who used it to communicate with his generals. `ROT-13 ("rotate by 13 places")` 
+	is a widely used example of a Caesar cipher where the shift is 13. 
+	In Python, the key for ROT-13 may be represented by means of the following dictionary:
+
+		key = {'a':'n', 'b':'o', 'c':'p', 'd':'q', 'e':'r', 'f':'s', 'g':'t', 'h':'u',
```
Comment (PabloVallejo) [inline 22.py]: Add space after colon `:`

File: 4_b.py [hunk 11]
```diff
@@ -1,13 +1,17 @@
...
-def vowel_or_not2():
+def vowel_or_not2 ():
+	
 	print ("\n Diference between vowel_or_not and vowel_or_not2 is that this function ask for user input:")
 	print ("\nEnter a letter to tell you if it is a vowel or not:")
-	vowel=input()
-	if (vowel=="a" or vowel=='e'or vowel=='i'or vowel=='o' or vowel=='u'):
+	
+	vowel = input()
+	if (vowel == "a" or vowel == 'e'or vowel == 'i'or vowel == 'o' or vowel == 'u'):
```
Comment (PabloVallejo) [inline 4_b.py]: Use `'....'` instead of `"..."`

File: 9.py [hunk 12]
```diff
@@ -1,21 +1,24 @@
...
-	print ("The list of values are:		",a)
-	i=0
-	s=len(a)
-	while i<s:							# While i<length of list
-		if ( a[i]==x ): return True			# if 'value' in position [i] == x return True
+	print ("\nThe value x is:			", x)
+	print ("The list of values are:		", a)
+	i = 0
+	s = len(a)
+	while i < s:						# While i<length of list
```
Comment (PabloVallejo) [inline 9.py]: Remove commented code.

File: 9.py [hunk 13]
```diff
@@ -25,9 +28,9 @@ def is_member(x,a):
 		else: return False					# then False
```
Comment (PabloVallejo) [inline 9.py]: Remove comment

File: 22.py (hunk 10 above)
Comment (PabloVallejo) [reply 22.py]: @miller9 please fix this line.

Comment (miller9) [reply 22.py]: Fixed it!

**Review by PabloVallejo — APPROVED** [review]

**Comment by PabloVallejo** [comment]
@miller9 I'm really happy with your progress! 😄

**Pull request merged**